core/src/main/java/apoc/meta/Meta.java

```diff
@@ -498,40 +492,45 @@ private void collectStats(SubGraph subGraph, Collection<String> labelNames, Coll
      * supports flexible sampling options, and does not scan the entire database.  The result is producing a table of
      * metadata that is useful for generating "Tables 4 Labels" schema designs for RDBMSs, but in a more performant way.
      */
     @Procedure
-    @Description("apoc.meta.nodeTypeProperties()")
-    public Stream<Tables4LabelsProfile.NodeTypePropertiesEntry> nodeTypeProperties(@Name(value = "config",defaultValue = "{}") Map<String,Object> config) {
-        MetaConfig metaConfig = new MetaConfig(config);
-        try {
-            return collectTables4LabelsProfile(metaConfig).asNodeStream();
-        } catch (Exception e) {
-            log.debug("meta.nodeTypeProperties(): Failed to return stream", e);
-            throw new RuntimeException(e);
+    @Description( "apoc.meta.nodeTypeProperties()" )
+    public Stream<Tables4LabelsProfile.NodeTypePropertiesEntry> nodeTypeProperties( @Name( value = "config", defaultValue = "{}" ) Map<String,Object> config )
+    {
+        MetaConfig metaConfig = new MetaConfig( config );
+        try
+        {
+            return collectTables4LabelsProfile( metaConfig ).asNodeStream();
+        }
+        catch ( Exception e )
+        {
+            log.debug( "apoc.meta.nodeTypeProperties(): Failed to return stream", e );
+            throw new RuntimeException( e );
         }
     }
 
     /**
-     * This procedure is intended to replicate what's in the core Neo4j product, but with the crucial difference that it
-     * supports flexible sampling options, and does not scan the entire database.  The result is producing a table of
-     * metadata that is useful for generating "Tables 4 Labels" schema designs for RDBMSs, but in a more performant way.
+     * This procedure is intended to replicate what's in the core Neo4j product, but with the crucial difference that it supports flexible sampling options, and
+     * does not scan the entire database.  The result is producing a table of metadata that is useful for generating "Tables 4 Labels" schema designs for
+     * RDBMSs, but in a more performant way.
      */
     @Procedure
-    @Description("apoc.meta.relTypeProperties()")
-    public Stream<Tables4LabelsProfile.RelTypePropertiesEntry> relTypeProperties(@Name(value = "config",defaultValue = "{}") Map<String,Object> config) {
-        MetaConfig metaConfig = new MetaConfig(config);
-        try {
-            return collectTables4LabelsProfile(metaConfig).asRelStream();
-        } catch (Exception e) {
-            log.debug("meta.relTypeProperties(): Failed to return stream", e);
-            throw new RuntimeException(e);
+    @Description( "apoc.meta.relTypeProperties()" )
+    public Stream<Tables4LabelsProfile.RelTypePropertiesEntry> relTypeProperties( @Name( value = "config", defaultValue = "{}" ) Map<String,Object> config )
+    {
+        MetaConfig metaConfig = new MetaConfig( config );
+        try
```

Comment:
nit: Neo4j formatting rules do not apply to the APOC project

Reply:
Question: Do you even have consistent formatting rules in APOC?

Reply:
![Screenshot 2021-05-28 at 17 45 46](https://user-images.githubusercontent.com/445792/120009707-89c48480-bfdc-11eb-87c7-f69f8b2bc497.png)

Only 2 results are returned for this coding style, so I'd at least avoid extra spaces within parentheses and newlines before curling braces.

We just filed https://github.com/neo4j-contrib/neo4j-apoc-procedures/issues/1936 to make this clearer for future contributors.